README.md

```diff
@@ -19,7 +19,11 @@ NostrKit can be used to publish events on a Nostr relay as well as request event
 
 ``` swift
 let keyPair = try KeyPair(privateKey: "<hex>")
-let relay = Relay(url: URL("<url>")!, onEvent: { print($0) })
+let relay = Relay(url: URL("<url>")!, 
+                  onEvent: { print($0) },
+                  onDisconnect: { error in 
+                      // Handle reconnecting here if needed
+                  })
```

Comment:
I'd leave the `onDisconnect` handler out of the code example to keep it as simple as possible for the readme. What do you think?

Reply:
Agree, will remove